code/debug/randomize.cpp

```diff
@@ -26,10 +26,10 @@ void createAndFillVector(int** v, unsigned int len) {
 }
 
 int main() {
-    int *v;
+    int *v = nullptr;
```

Comment:
I propose to initialize the pointer. Many students inspected the crash inside `swap` in the debugger and nothing looked wrong to them. If we make `v` `nullptr` initially, then the addresses inside `swap` will look more suspicious. Also the example crashes more deterministically.

Reply:
:+1: